### PR TITLE
Add changelog doc

### PR DESCRIPTION
**Pull request by YQBird**
# Pull request

### What kind of change does this PR introduce?
_This PR includes a new CHANGELOG.md file to record all notable change for this project_

### Did you add tests for your changes?
_No_

### Summary
_A new CHANGELOG.md file will help to record all the change for this project in the future. Please remember to change the CHANGELOG.md file and update the version every time a contributor make a PR. The plugin user can easily find information from this document._

### Other information
_Issue #26. The changelog format is from this site http://keepachangelog.com/en/0.3.0/._

### CODE REVIEW

**Comment by YQBird** [comment]
Hi @iGitScor. Please let me know if there is any sentence or word which is not written well. I'm not a native English speaker but I will work on that. By the way, do you edit the logo and gif image by your self? That's very cool!

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# Change log
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com) and this project adheres to [Semantic Versioning](http://semver.org).
```
Comment (iGitScor) [inline CHANGELOG.md]: I would rather have written
`this project follows the Semantic Versioning code`

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+- Update package dependencies for eslint and eslint plugin.
+
+## 0.2.2 - 2016-12-19
+### Changed
+- Update eslint package version.
+- Fix bugs, checking the files exist before reading files.
+
+## 0.2.1 - 2016-12-16
+### Changed
+- Divide the logic of manual call to replace content.
```
Comment (iGitScor) [inline CHANGELOG.md]: This modification was made for this reason:
the plugin can be called outside a webpack workflow, that's why I extracted replace method to not call apply method.

Could you replace this change entry by:
`Extract replace method to be called outside a webpack workflow` ?

**Review by iGitScor — APPROVED** [review]
Wow, nice job ! You rock 👍

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
+# Change log
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com) and this project adheres to [Semantic Versioning](http://semver.org).
+
+## 0.3.6 - 2017-03-04
+### Added
+- Add CHANGELOG.md.
```
Comment (iGitScor) [inline CHANGELOG.md]: Could you add `Update contribution documentation` ? Please, thank you

Comment (YQBird) [reply CHANGELOG.md]: Sure. I will update all these change today. Had some personal stuff for the past two day.

Comment (iGitScor) [reply CHANGELOG.md]: No pressure, personal stuffs come before open source 😉

**Comment by YQBird** [comment]
Hi @iGitScor. Just pushed a new commit to update the change. Please let me know any concern you have. Thanks!

**Comment by iGitScor** [comment]
Perfect contribution, thank you @YQBird

**Comment by YQBird** [comment]
Cool, Thanks!

**Pull request merged**